bin/cli.js

```diff
@@ -108,16 +108,25 @@ if (typeof process === 'undefined') {
 // fall back to spec by default
 var reporter = 'spec'
 
+// Built-in Mocha reporters that are compatible
+var mochaReporters = ['spec', 'xunit', 'json'];
```

Comment:
The comment is a bit misleading as most all built-in reporters are compatible, like `dot`, `min` that are not included in this list. 

I'd just change the comment mentioning CasperJS's patched require and a link to the issue.